TrackletAlgorithm/MemoryTemplate.hh

```diff
@@ -54,7 +54,7 @@ public:
 	return nentries_[bx];
   }
 
-  DataType* get_mem() {return dataarray_;}
+  const DataType (&get_mem() const)[1<<NBIT_BX][1<<NBIT_ADDR] {return dataarray_;}
```

Comment:
was the change to a reference required or just the addition of the size of the arrays?

Reply:
It's needed to allow the latency pragma to operate on the return value of the method. The pragma wants an lvalue, while a pointer is an rvalue.